modules/node_modules/@frogpond/ccc-wpjson-feed/index.js

```diff
@@ -53,3 +53,11 @@ export function convertWpJsonItemToStory(item) {
 		title: JSDOM.fragment(item.title.rendered).textContent.trim(),
 	}
 }
+
+export function deprecatedWpJson() {
+	return [{
+		content: "",
```

Comment:
Should we add a link to the App Store entry / GitHub page… maybe a GitHub issue where we describe the message?

Reply:
I like the idea of a GitHub issue link!

Reply:
Server repo, I think